packages/nextjs-template/components/DendronLayout.tsx

```diff
@@ -133,27 +135,32 @@ export default function DendronLayout(
         className="site-layout"
         style={{
           marginTop: 64,
-          flexDirection: "row",
         }}
       >
-        <Layout
-          className="site-layout-sidebar"
-          style={{
-            flex: "0 0 auto",
-            width: `calc((100% - ${LAYOUT.BREAKPOINTS.lg}) / 2 + ${
-              isCollapsed ? SIDER.COLLAPSED_WIDTH : SIDER.WIDTH
-            }px)`,
-            paddingLeft: `calc((100% - ${LAYOUT.BREAKPOINTS.lg}) / 2)`,
-            minWidth: isCollapsed ? SIDER.COLLAPSED_WIDTH : SIDER.WIDTH,
-          }}
-        >
-          {sidebar}
-        </Layout>
-        <Layout
-          className="side-layout-main"
-          style={{ maxWidth: LAYOUT.CONTENT_MAX_WIDTH }}
-        >
-          {content}
+        <DendronNotice show={getStage() === "dev"}>
```

Comment:
can you change the text to the following? (NOTE: the link won't resolve right now because we haven't updated it yet)

```
NOTE: Pages are [dynamically compiled in local preview](https://wiki.dendron.so/notes/yYMuhi2TmTC63MysmtwqH.html#navigating-pages-is-slow-for-local-preview) and will take a second to load. 
```

Reply:
perhaps we could add a short link redirect.